cicd/build.sh

```diff
@@ -200,8 +200,8 @@ if [ "$DEV" == true ]; then
     install_py_req req_dev.txt
 
     # Solving numpy libgfortran-ed201abd.so.3.0.0 dependency problem
```

Comment:
Add this comment line below where we have moved the numpy related code

Reply:
Done with changes